arches/app/utils/data_management/resources/formats/csvfile.py

```diff
@@ -423,9 +423,8 @@ def write_resources(self, graph_id=None, resourceinstanceids=None, **kwargs):
         csvs_for_export = []
 
         tiles = self.group_tiles(
-            self.tiles.order_by(
-                "nodegroup_id"
-            ).values(),  # TODO: refactor to avoid going to the db again
+            # TODO: refactor to avoid going to the db again
+            self.tiles.order_by("nodegroup_id", "sortorder").values(),
```

Comment:
Behavior change: shouldn't this sort by sort order? Added. Can pull this out into a separate PR if preferable.